RadixWallet/Core/SharedModels/P2P/ConnectorExtension/P2P+ConnectorExtension+Response.swift

```diff
@@ -78,11 +80,11 @@ extension P2P.ConnectorExtension.Response.LedgerHardwareWallet {
 		case deriveAndDisplayAddress(DerivedAddress)
 
 		public struct GetDeviceInfo: Sendable, Hashable, Decodable {
-			public let id: HexCodable32Bytes
+			public let id: Exactly32Bytes
```

Comment:
`HexCodable32Bytes` has been removed, and replaces by: `Exactly32Bytes`, which has some "siblings":

- `Exactly29Bytes`
- `Exactly33Bytes`
- `Exactly64Bytes`
- `Exactly65Bytes`

[From Sargon, all declared with a single macro call](https://github.com/radixdlt/sargon/blob/develop/src/core/types/exactly_n_bytes.rs#L507-L549)